src/domain/entities/DataSetPeriodDate.ts

```diff
@@ -0,0 +1,10 @@
+import { PeriodDate } from "$/domain/entities/PeriodDate";
+import { Id } from "$/domain/entities/Ref";
+
+export type DataSetPeriodDate = {
+    id: Id;
+    name: string;
+    periodDate: PeriodDate;
+    outcomeDate: PeriodDate;
+    outputDate: PeriodDate;
+};
```

Comment:
[no need to change]

I see you have create an entity+repo for DataSetPeriodDate. This is something we have been discussing lately, and I'd confirm it with @xurxodev but from our last chat about this, he stated that having different "flavors" or entities is not correct. I see two alternatives: 

- Add 3 fields would be in DataSet. I think this is the most simple option. And you we simply improve on our DataSet/DataSetRepository. You may then say that we typically don't need them and getting the info from the attribute is very costly (i don't think it's the case here). Then we'd say alternatives there
-Have a DataSetPeriodDate, but not as flavor, but as its own entity. So having `dataSetId: Id` + 3 fields. Typically we do this if the entity has sense in isolation.